backend/src/main/java/de/cae/XYFleet/ressource/BookingRessource.java

```diff
@@ -1,27 +1,24 @@
 package de.cae.XYFleet.ressource;
 
-import org.jooq.Result;
-import org.jooq.Record;
 import org.jooq.codegen.XYFleet.tables.records.BookingsRecord;
-import org.jooq.codegen.XYFleet.tables.records.UsersRecord;
 import org.restlet.resource.*;
 
 
 import static de.cae.XYFleet.authentication.XYAuthorizer.ROLE_SECURITY;
 import static de.cae.XYFleet.authentication.XYAuthorizer.ROLE_USER;
 import static org.jooq.codegen.XYFleet.Tables.BOOKINGS;
-import static org.jooq.codegen.XYFleet.Tables.USERS;
 
-public class Booking extends XYServerResource {
+public class BookingRessource extends XYServerResource {
     private int bookingIdentifier;
 
     @Get("txt")
-    public String toString() {
+    public String toString() throws ResourceException {
         checkInRole(ROLE_SECURITY);
         //SELECT * FROM bookings where id = {bookingIdentifier}
         BookingsRecord result = dslContext.fetchOne(BOOKINGS, BOOKINGS.ID.eq(bookingIdentifier));
 
-        return result != null? result.formatJSON(jSONFormat) : null;
```

Comment:
Exhibit A: Missing ' '

Reply:
Where is a space missing??? please specifiy

Reply:
Before the '?' - just usw Autoformat.